src/test/fuzz/addrman.cpp

```diff
@@ -326,4 +307,4 @@ FUZZ_TARGET_INIT(addrman_serdeser, initialize_addrman)
     data_stream << addr_man1;
     data_stream >> addr_man2;
     assert(addr_man1 == addr_man2);
-}
+}
```

Comment:
It'd be good to avoid making unrelated changes like this (deleting the file ending newline).